notebooks/book2/11/mcmc_gmm_demo.ipynb

```diff
@@ -1,9 +1,17 @@
 {
```

Comment:
<p>Is <code>tau</code> precision or scale? I think this may be useful to clarify and also add such minimal code comments. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>As of now I have added a note that <code>tau</code> is a scaling factor in proposal distribution  </p>